packages/legend-application-repl/src/components/dataCube/grid/DataCubeGrid.tsx

```diff
@@ -55,6 +55,8 @@ export const DataCubeGrid = observer((props: { editorStore: REPLStore }) => {
         alwaysMultiSort={true}
         rowModelType="serverSide"
         serverSideDatasource={dataCubeState.grid.clientDataSource}
+        suppressAggFuncInHeader={true}
+        suppressServerSideInfiniteScroll={true}
```

Comment:
what's the point of this flag, can you comment on it?

Reply:
per this thread, wouldn't it mess up by enabling client-side sorting?
https://github.com/ag-grid/ag-grid/issues/6963